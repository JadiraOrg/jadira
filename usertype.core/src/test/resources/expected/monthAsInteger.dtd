<!ELEMENT dataset (
    MONTHASINTEGER*)>

<!ELEMENT MONTHASINTEGER EMPTY>
<!ATTLIST MONTHASINTEGER
    ID CDATA #REQUIRED
    MONTH CDATA #IMPLIED
    NAME CDATA #IMPLIED
>

