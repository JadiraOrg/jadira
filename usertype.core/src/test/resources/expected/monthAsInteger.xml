<?xml version='1.0' encoding='UTF-8'?>
<dataset>
  <monthAsInteger ID="0" MONTH="1" NAME="test_0"/>
  <monthAsInteger ID="1" MONTH="2" NAME="test_1"/>
  <monthAsInteger ID="2" MONTH="12" NAME="test_2"/>
</dataset>